### ISSUE TITLE
Add Back-to-Top button for long pages

### ISSUE THREAD

**Issue by 2024itb047samata** [aditiraj2006/KrishiSetu#21]
Description:
The website currently lacks a quick navigation option for long pages. Adding a floating “Back to Top” button would improve usability.

Suggested Changes
Add floating button at bottom-right
Show button only after scrolling down
Smooth scroll to top on click
